x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_enrichment/page_content.tsx

```diff
@@ -82,20 +82,38 @@ export function StreamDetailEnrichmentContent({
     return <RootStreamEmptyPrompt />;
   }
 
+  const isNonAdditiveSimulation = simulation && simulation.is_non_additive_simulation;
+  const isSubmitDisabled = Boolean(!hasChanges || isNonAdditiveSimulation);
+
+  const confirmTooltip = isNonAdditiveSimulation
+    ? {
+        title: i18n.translate('xpack.streams.streamDetailView.nonAdditiveProcessorsTooltip.title', {
```

Comment:
Nit: the other i18n IDs here have `managementTab.enrichment` here also to be more specific.